src/Gui/RadiumWindow/SimpleWindow.cpp

```diff
@@ -60,9 +62,9 @@ void SimpleWindow::addRenderer( const std::string&,
     m_viewer->addRenderer( e );
 }
 
-void SimpleWindow::prepareDisplay( const std::string& ) {
+void SimpleWindow::prepareDisplay( const std::string& sceneName ) {
+    if ( !sceneName.empty() ) { statusBar()->showMessage( sceneName.c_str() ); }
```

Comment:
I still don't get the point here ... why simple window display this message ? should not be the code as simple as possible without any superflous stuff ?

Reply:
I recall the objective of this PR that is to allow users to build scene without worry about the renderer that will display it.
So, I will definitively not refactor in this PR how the apps interact with the scene after loading a file or creating a scene from scratch. So, the sceneName parameter will remain and is not used in SimpleWindow as before. If this is problematic for you, fell free to either contribute to this PR or propose another PR later.